exercises/isogram/src/example/java/Isogram.java

```diff
@@ -0,0 +1,29 @@
+package exemple;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Isogram {
+	
+	public static boolean isIsogram(String word){
+		
+		Set<Character> charset= new HashSet<Character>();
+		String[] words = word.split(" ");
+		String newword = concat(words);
+		words = newword.split("-");
+		newword = concat(words).toLowerCase();
+		
+		for(int i = 0; i < newword.length(); i++)
+			charset.add(newword.charAt(i));
+		
+		return charset.size() == newword.length();
+	}
+	
+	public static String concat(String[] words){
+		String newword = new String("");
+		for(int i = 0; i < words.length; i++)
+			newword = newword + words[i];
```

Comment:
If you like, you could replace this with:
```Java
String newWord = "";
for (String word: words){
    newWord += word;
}
```
No need for the index variable.

Or, for some extra-delicious Java 8:

```Java
String newWord = stream(words).collect(joining());
``` 

To use the `Streams` version you'll need to import:

```Java
import static java.util.Arrays.stream;
import static java.util.stream.Collectors.joining;
```